### PR TITLE
pinephone-kernel: update to 5.10.12.

### PR DESCRIPTION
**Pull request by jjsullivan5196**
#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

New version of megi's pinephone kernel. In addition to regular kernel updates, this brings new improvements:

- Display refresh is now at 60hz from ~45hz
- Improved wifi power management
- Modem driver has added support for new Quectel modem features
- Audio codec drivers work better with the hardware, higher sample rates in phone calls. This also unblocks #25084 

So far I've tested a pretty barebones userspace on top of this, sway window manager, firefox, mpv music/video playback, wifi/bluetooth, and typec docking are all still working.

I also cleaned the template up a bit, to better match the mainline kernel. Of course it's not exactly 1:1, but I believe the style is close enough.

Also, for the maintainers: I admit I've been pretty lazy about documentation. I'd like to get some docs going for mklive and sxmo on the device, so we at least have a MVP userspace people can try out (or try and help some other enthusiastic people do this). Of course, this PR will make it easier to get there :-)

### CODE REVIEW

File: srcpkgs/pinephone-kernel/template [hunk 1]
```diff
@@ -41,37 +41,49 @@ mutable_files="
...
 
 	# Always use our revision to CONFIG_LOCALVERSION to match our pkg version.
 	sed -i -e "s|^\(CONFIG_LOCALVERSION=\).*|\1\"_${revision}\"|" .config
 }
 
 do_build() {
-	local arch _cross _args
-	_args="Image modules dtbs"
+	local arch
 	arch=arm64
```
Comment (ericonr) [inline srcpkgs/pinephone-kernel/template:82]:
```suggestion
	local arch=arm64
```

**Comment by ericonr** [comment]
Can you test if iwd also works? It wasn't working for a while on pinebookpro.

File: srcpkgs/pinephone-kernel/template (hunk 1 above)
Comment (jjsullivan5196) [reply srcpkgs/pinephone-kernel/template]: Got it :+1:

**Comment by jjsullivan5196** [comment]
@ericonr sorry for the delay :^P `iwd` just works! (tm)

I was screwing around for a couple of hours trying to get my usb networking script to work, turns out networkmanager was interfering somehow. Switched over to iwd, got wifi and ssh over usb working at once :^)

File: srcpkgs/pinephone-kernel/template [hunk 2]
```diff
@@ -1,15 +1,15 @@
 # Template file for 'pinephone-kernel'
 pkgname=pinephone-kernel
-version=5.8.10
+version=5.10.12
 revision=1
-_commit=b8647e317c32f64543c32465abb7b4651e6e3199
-wrksrc="linux-${_commit}"
+_tag=orange-pi-5.10-20210130-2109
```
Comment (ericonr) [inline srcpkgs/pinephone-kernel/template:5]: Why is it this weird tag?

Comment (jjsullivan5196) [reply srcpkgs/pinephone-kernel/template:5]: The author uses an `orange-pi-{kernver}` branch to release all their embedded patches, including the pinephone. This tag references the latest release on the 5.10 branch. The `orange-pi-...` branch recommended for use by the author https://xnux.eu/devices/pine64-pinephone.html#toc-how-my-kernel-tree-is-updated

**Pull request merged**